components/common/homepage/extra/testimonial.jsx

```diff
@@ -0,0 +1,196 @@
+import Slider from 'react-slick'
+import styles from '../comment.module.css'
+import Image from 'next/image'
+import { Col, Row } from 'react-bootstrap'
+
+const Testimonials = () => {
+  const settings = {
+    infinite: true,
+    slidesToShow: 3,
+    slidesToScroll: 1,
+    vertical: true,
+    verticalSwiping: true,
+    autoplay: true,
+  }
```

Comment:
Well done on the change implementation. Please note that the FAQ and testimonies should be in different sections stacked on each other. 

![image](https://user-images.githubusercontent.com/10332499/203934116-cc7c5d46-dbf6-43cc-b4d7-cfb194d5a2e7.png)